hooks/playbooks/ironic_network.yml

```diff
@@ -15,6 +15,7 @@
     _provider_physical_network: ironic
     _provider_network_type: flat
     _availability_zone_hints: null  # Comma separated list of strings
+    _create_provisioning_router: true  # Set to false to opt out of router creation
```

Comment:
so again: why you make parameter for it if you can not overwrite it?
All the vars does not make sense and should be "converted" to "default" value.